content/guides/references/configuration.md

```diff
@@ -224,28 +224,36 @@ You can override configuration for either the E2E or
 Component Testing specific viewports in configuration file (`cypress.json` by
 default):
 
-```json
+:::cypress-config-example
```

Comment:
Let's not make any changes to `configuration.md` in this PR, since it's going to be moved to a "legacy configuration" page

Reply:
🤦‍♂️ whoops. that makes sense.